apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/RestResourceBase.java

```diff
@@ -203,9 +267,22 @@ protected static byte[] decodeBase64(String base64encoded) {
 
   // // // Helper methods for JAX-RS response construction
 
-  protected static Response restServiceError(Response.Status httpStatus, String failMessage) {
-    return Response.status(httpStatus)
-        .entity(new ApiError(failMessage, httpStatus.getStatusCode()))
-        .build();
+  protected static Uni<RestResponse<Object>> apiErrorResponseUni(
+      Response.Status httpStatus, String failMessage) {
+    return Uni.createFrom().item(apiErrorResponse(httpStatus, failMessage));
+  }
+
+  protected static RestResponse<Object> apiErrorResponse(
+      Response.Status httpStatus, String failMessage) {
+    return RestResponse.status(httpStatus, new ApiError(failMessage, httpStatus.getStatusCode()));
+  }
+
+  protected static RestResponse<Object> restResponseCreatedWithName(String createdName) {
+    return RestResponse.status(
+        Response.Status.CREATED, Collections.singletonMap("name", createdName));
+  }
+
+  protected static RestResponse<Object> restResponseOkWithName(String name) {
+    return RestResponse.status(Response.Status.OK, Collections.singletonMap("name", name));
```

Comment:
Where is this used, becuase this will not be documented correctly in the openapi? above as well?

Reply:
I can have a look at this, I don't remember off-hand.